omnibus/files/private-chef-cookbooks/private-chef/recipes/nginx.rb

```diff
@@ -125,6 +130,7 @@
  'routes.lua',
  'dispatch_route.lua',
  'check_token_using_endpoint.lua',
+ 'check_token_using_endpoint_required_recipe.lua',
```

Comment:
I thought about that, but all of them have very different post validation actions so making it generic might not be all that useful. Furthermore, I uncovered a bug with the validation endpoint where it'll return a 200 for _any_ properly formatted `GET` request. I'm working to resolve that issue before this PR moves ahead. That issue likely impacts the compliance profile endpoint as well.

```
root@api:/tmp# !curl
curl -kLI -X GET https://api.chef-server.dev/organizations/test/validate/i_like_bananas
HTTP/1.1 200 OK
Server: openresty/1.11.2.1
Date: Wed, 05 Apr 2017 17:35:22 GMT
Content-Type: text/html
Content-Length: 2474
Last-Modified: Tue, 04 Apr 2017 22:40:36 GMT
Connection: keep-alive
ETag: "58e420e4-9aa"
Accept-Ranges: bytes
```